include/dca/phys/models/traits.hpp

```diff
@@ -38,10 +38,10 @@ struct has_non_density_interaction<FeAsLattice<BaseLattice>> {
   constexpr static bool value = true;
 };
 
-//template <class PointGroup>
-//struct has_non_density_interaction<TwobandCu<PointGroup>> {
-//  constexpr static bool value = true;
-//};
+template <class PointGroup>
+struct has_non_density_interaction<TwobandCu<PointGroup>> {
```

Comment:
This is a special naming case we haven't considered for now I think we should leave it alone. It is really a type but its either a query function or a "variable" as a metaprogramming entity.

Reply:
actually we can make it a real variable with c++17. Will update.

Reply:
done in bc0b2c91a9581e3591c52157db8bd1de90312ae9